doc/manual/en/config/url-params.rst

```diff
@@ -104,7 +104,7 @@ also a ``backend`` parameter in the config to fix this.
 .. code::
 
     Default: cgi-bin (backend=cgi-bin)
-    Options: cgi-bin (backend=cgi-bin), oh (backend=oh), oh2 (backend=oh2)
+    Options: cgi-bin (backend=cgi-bin), oh (backend=oh), oh2 (backend=oh2), mqtt
```

Comment:
we should go with "openhab" here, all the other variants are currently kept for backwards compability and should be removes with the next library upgrade